### DISCUSSION TITLE
过期续费的支付方式问题

### DISCUSSION THREAD

**Discussion by kidokogills** [getlantern/lantern-forum-cn#814] [Q&A]
已经过期很多天了，一直没办法续费，银行卡不支持，也没收到其他支付方式的提供，一直也没有得到回应，我也就是想续个费而已，为何如此困难？

**Comment by MaxMKMax** [comment]
@kidokogills 您好. 请您提供我您的账号邮箱. 我让支付团队直接给您发支付链接.
